servers/linkedin-mcp-server/server.yaml

```diff
@@ -19,6 +19,6 @@ source:
 config:
   description: Configure LinkedIn authentication using your session cookie
   secrets:
-    - name: linkedin-mcp-server.linkedin_cookie
+    - name: LinkedIn li_at cookie value
```

Comment:
This is a reference key, not really a description, so you'll want to keep this as is. Maybe you could throw it in `example?`

Reply:
No, this might be some caching issue. 

When I test this PR, I get this result:
<img width="433" height="271" alt="Screenshot 2025-07-14 at 12 49 01" src="https://github.com/user-attachments/assets/d5893be8-bc87-4ef2-9589-f0d1a25758e3" />

And while you can set the value, the value is not stored correctly.

This is how the `secrets` block should look:

```
secrets:
    - name: linkedin-mcp-server.cookie
      env: LINKEDIN_COOKIE
      example: AQETADxR7bsCqpZlAAACm-lNHKAAA...
```